config/action.d/mail-whois-lines.conf

```diff
@@ -42,7 +42,7 @@ actionban = printf %%b "Hi,\n
             Here is more information about <ip>:\n
             `whois <ip> || echo missing whois program`\n\n
             Lines containing IP:<ip> in <logpath>\n
-            `grep -E '(^|[^0-9])<ip>([^0-9]|$)' <logpath>`\n\n
+            `grep -E <grepopts> '(^|[^0-9])<ip>([^0-9]|$)' <logpath>`\n\n
```

Comment:
On Sun, 26 Jul 2015, Viktor Szépe wrote:
>  > @@ -42,7 +42,7 @@ actionban = printf %%b "Hi,\n
>  >              Here is more information about <ip>:\n
>  >              `whois <ip> || echo missing whois program`\n\n
>  >              Lines containing IP:<ip> in <logpath>\n
>  > -            `grep -E '(^|[^0-9])<ip>([^0-9]|$)' <logpath>`\n\n
>  > +            `grep -E <grepopts> '(^|[^0-9])<ip>([^0-9]|$)' <logpath>`\n\n

>      (is it? ;) ) mandatory
>    Yes, | and ( otherwise would need escaping.

THAT IS RIGHT! ;)